mypy/semanal_namedtuple.py

```diff
@@ -262,16 +262,17 @@ def parse_namedtuple_args(self, call: CallExpr, fullname: str
 
         Return None if the definition didn't typecheck.
         """
+        type_name = 'NamedTuple' if fullname == 'typing.NamedTuple' else 'namedtuple'
         # TODO: Share code with check_argument_count in checkexpr.py?
         args = call.args
         if len(args) < 2:
-            self.fail("Too few arguments for namedtuple()", call)
+            self.fail('Too few arguments for "{0}()"'.format(type_name), call)
```

Comment:
Style nit: We tend to use just `{}` in format strings instead of `{0}` whenever possible (applies to here and elsewhere it this PR). Not a big deal though.